src/Pages/HumansAtNasa/HumansAtNasa.tsx

```diff
@@ -1,16 +1,17 @@
-﻿﻿import React from "react";
+import {HumansInSpace} from "../../Components/HumansInSpace/HumansInSpace";
+import React from "react";
 import {HumanAtNasa} from "../../Components/HumanAtNasa/HumanAtNasa";
 import './HumansAtNasa.scss';
 
 export function HumansAtNasa() {
     return (
-        <section className='humansAtNasaBody'>
-            <h1>Humans At Nasa</h1>
+        <div className='humansAtNasaBody'>
+            <h1 className='opportunity'>Humans At Nasa</h1>
             <p>NASA's Mars missions are designed, built and operated by some of the best and brightest. Meet some of the amazing people who do their best work on another planet.</p>
+            <HumansInSpace/>
```

Comment:
Is this in the right place?
Seems like it should have its own section or something rather than being in the middle of the Employees of NASA section?

Reply:
We discussed creating a modal, but given time constraints we thought we would make a box at the top of the employees page instead